File: Backend (Springboot)/Annapurna/src/main/java/com/annapurna/dto/OrderResponseDTO.java
package com.annapurna.dto;

import com.annapurna.domain.OrderStatus;
import com.annapurna.domain.PaymentStatus;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class OrderResponseDTO {
    private Long orderId;
    private Long tiffinId;
    private String tiffinName;
    private int quantity;
    private double totalPrice;
    private OrderStatus status;
    private PaymentStatus paymentStatus; // ✅ added
    private LocalDateTime orderTime;
    private String razorpayOrderId;      // ✅ optional but useful
}

File: Backend (DNET)/Annapurna/DTOs/OrderResponseDTO.cs
using AnnapurnaAPI.Models;

namespace AnnapurnaAPI.DTOs
{
    public class OrderResponseDTO
    {
        public long OrderId { get; set; }     // changed
        public long TiffinId { get; set; }    // changed
        public string TiffinName { get; set; } = string.Empty;
        public int Quantity { get; set; }
        public double TotalPrice { get; set; }
        public OrderStatus Status { get; set; }
        public PaymentStatus PaymentStatus { get; set; }
        public DateTime OrderTime { get; set; }
        public string? RazorpayOrderId { get; set; }

        public OrderResponseDTO() { }

        public OrderResponseDTO(long orderId, long tiffinId, string tiffinName, int quantity, double totalPrice, OrderStatus status, PaymentStatus paymentStatus, DateTime orderTime, string? razorpayOrderId)
        {
            OrderId = orderId;
            TiffinId = tiffinId;
            TiffinName = tiffinName;
            Quantity = quantity;
            TotalPrice = totalPrice;
            Status = status;
            PaymentStatus = paymentStatus;
            OrderTime = orderTime;
            RazorpayOrderId = razorpayOrderId;
        }
    }
}
